src/clipboard_handlers/data_validation_clipboard.ts

```diff
@@ -84,40 +95,43 @@ export class DataValidationClipboardHandler extends AbstractCellClipboardHandler
   }
 
   private pasteDataValidation(
-    origin: CellPosition,
+    origin: Maybe<ClipboardDataValidationRule>,
     target: CellPosition,
     isCutOperation?: boolean
   ) {
-    const rule = this.getters.getValidationRuleForCell(origin);
-    if (!rule) {
-      const targetRule = this.getters.getValidationRuleForCell(target);
-      if (targetRule) {
-        // Remove the data validation rule on the target cell
-        this.adaptDataValidationRule(target.sheetId, targetRule, [], [positionToZone(target)]);
-      }
-      return;
-    }
-    const zone = positionToZone(target);
-    for (const range of rule.ranges) {
-      if (isInside(origin.col, origin.row, range.zone)) {
-        const toRemoveZone: Zone[] = [];
-        if (isCutOperation) {
-          toRemoveZone.push(positionToZone(origin));
+    if (origin) {
+      const zone = positionToZone(target);
+      const rule = this.getters.getValidationRuleForCell(origin.position) || origin.rule;
```

Comment:
Could you remind me the reason why we use the current validation rule and not `origin.rule` ? I think it deserves a comment in the code

Edit: it can be removed (see comment below)
```suggestion
      const rule = origin.rule;
```